strutil/strutil_test.go

```diff
@@ -225,3 +225,67 @@ func (strutilSuite) TestEllipt(c *check.C) {
 		c.Check(strutil.ElliptLeft(t.in, t.n), check.Equals, t.left, check.Commentf("%q[-%d:] -> %q", t.in, t.n, t.left))
 	}
 }
+
+func (strutilSuite) TestListsSame(c *check.C) {
+	tt := []struct {
+		l1  []string
+		l2  []string
+		exp bool
+	}{
+		{
+			[]string{},
+			[]string{},
+			true,
+		},
+		{
+			[]string{},
+			nil,
+			true,
+		},
+		{
+			[]string{"a"},
+			nil,
+			false,
+		},
```

Comment:
For completeness, I think it'd be useful to add  `{nil, nil, true}` test case too.

Reply:
Add the test case